Security/Authentication/Token/OAuth2TokenFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Security\Authentication\Token;
+
+use Psr\Http\Message\ServerRequestInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class OAuth2TokenFactory
```

Comment:
@rjwebdev Can we make the class `final`?

Reply:
I made the class final. In my opinion, I don't think classes used in a bundle should be made final since there's always a chance someone who's using the  bundle, wants to override pieces of it